### PR TITLE
[DOC] print and clip api, test=develop

### PR DESCRIPTION
**Pull request by qili93**
Change the code samples of the following API:

- paddle.static.Print
- paddle.nn.GradientClipByValue
- paddle.nn.GradientClipByNorm
- paddle.nn.GradientClipByGlobalNorm

Corresponding Paddle PR: https://github.com/PaddlePaddle/Paddle/pull/27670

### CODE REVIEW

**Comment by qili93** [comment]
image

image

image

image

**Comment by qili93** [comment]
image
image
image
image

File: doc/paddle/api/paddle/fluid/layers/control_flow/Print_cn.rst [hunk 1]
```diff
@@ -37,35 +37,20 @@ Print
 
 .. code-block:: python
 
-    import paddle.fluid as fluid
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/layers/control_flow/Print_cn.rst:40]: 上面的Variable -> Tensor 
不用单独写返回类型~

File: doc/paddle/api/paddle/fluid/clip/GradientClipByValue_cn.rst [hunk 2]
```diff
@@ -26,72 +25,75 @@ GradientClipByValue
  - **min** (float，optional) - 要修剪的最小值。如果用户没有设置，将被自动设置为 ``-max`` （此时 ``max`` 必须大于0）。
  - **need_clip** (function, optional) - 类型: 函数。用于指定需要梯度裁剪的参数，该函数接收一个 ``Parameter`` ，返回一个 ``bool`` (True表示需要裁剪，False不需要裁剪)。默认为None，此时会裁剪网络中全部参数。
   
-**代码示例1：静态图**
+**代码示例1：动态图**
```
Comment (TCChenlong) [inline doc/paddle/api/paddle/fluid/clip/GradientClipByValue_cn.rst]: 代码改动同英文~

Comment (qili93) [reply doc/paddle/api/paddle/fluid/clip/GradientClipByValue_cn.rst]: Done

**Review by hong19860320 — APPROVED** [review]
LGTM

File: doc/paddle/api/paddle/fluid/layers/control_flow/Print_cn.rst (hunk 1 above)
Comment (qili93) [reply doc/paddle/api/paddle/fluid/layers/control_flow/Print_cn.rst:40]: 确认过，静态图还是Variable，不用修改

Comment (qili93) [reply doc/paddle/api/paddle/fluid/layers/control_flow/Print_cn.rst:40]: 返回类型已删

**Review by TCChenlong — APPROVED** [review]
LGTM

**Pull request merged**